bic

```diff
@@ -366,7 +366,12 @@ build_feed() {
 build_static() {
   [[ ! -d "${SRC_DIR}"/static ]] && return
   log "Building /static -> /"
-  cp -a "${SRC_DIR}"/static/. "${DEST_DIR}"
+  # Archive by default
+  local args=(-a)
+  # And run interactively so as to not clobber files unless explicitly disabled
+  [[ -z "${BIC_OVERWRITE:-}" ]] && args+=(-i)
+  # shellcheck disable=SC2216
+  echo n | cp "${args[@]}" "${SRC_DIR}"/static/. "${DEST_DIR}"
```

Comment:
don't love this, but wasn't able to immediately come up with another way to accomplish it https://www.shellcheck.net/wiki/SC2216